include/zigbee/zigbee_zcl_scenes.rst

```diff
@@ -0,0 +1,33 @@
+.. _lib_zigbee_zcl_scenes:
+
+Zigbee ZCL scene helper
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Zigbee ZCL scene helper library provides a set of macros that you can use to define the scene cluster endpoint and table size properties when working with the ZCL scenes.
+
+.. _lib_zigbee_zcl_scenes_options:
+
+Configuration
+*************
+
+To enable the Zigbee ZCL scene helper library, set the :option:`CONFIG_ZIGBEE_SCENES` Kconfig option.
+Setting this option allows you to configure the following library-specific Kconfig options:
+
+* :option:`CONFIG_ZIGBEE_SCENES_ENDPOINT` - This option sets the value for the endpoint for Zigbee scenes.
+* :option:`CONFIG_ZIGBEE_SCENE_TABLE_SIZE` - This options sets the value for the scene table size.
+
+To configure the logging level of the library, use the :option:`CONFIG_ZIGBEE_SCENES_LOG_LEVEL` Kconfig option.
+
+API documentation
+*****************
+
+| Header file: :file:`include/zigbee/zigbee_zcl_scenes.h`
+| Source file: :file:`subsys/zigbee/lib/zigbee_scenes/zigbee_zcl_scenes.c`
+
+.. doxygengroup:: zigbee_logger_ep
```

Comment:
@tomchy , the header is not updated for doxygen. Can you help?

Reply:
Fixed!